evm/src/omni-bridge/contracts/BridgeTypes.sol

```diff
@@ -44,7 +44,8 @@ library BridgeTypes {
         string token,
         string name,
         string symbol,
-        uint8 decimals
+        uint8 decimals,
+        uint8 originDeciamls
```

Comment:
should it be `originDecimals`?

Reply:
Fixed in 862fe1641646cd764427cc09d04121db5f4447b6

Reply:
@karim-en Hmm, I see a fix, but you made a typo in new commit `Deciamls` -> `Decimals`

Reply:
80602581078d32653ad061dbecfa35d4a552c5d2